tests/unittest_brain_numpy_core_numeric.py

```diff
@@ -62,5 +64,26 @@ def test_numpy_function_calls_inferred_as_ndarray(self):
                 )
 
 
+@pytest.mark.skipif(not HAS_NUMPY, reason="This test requires the numpy library.")
+@pytest.mark.parametrize(
+    "method, expected_args",
+    [
+        ("zeros_like", ["a", "dtype", "order", "subok", "shape"]),
+        ("full_like", ["a", "fill_value", "dtype", "order", "subok", "shape"]),
+        ("ones_like", ["a", "dtype", "order", "subok", "shape"]),
+        ("ones", ["shape", "dtype", "order"]),
+    ],
+)
+def test_parameters(method, expected_args) -> None:
```

Comment:
```suggestion
def test_function_parameters(method: str, expected_args: List[str]) -> None:
```

Reply:
Why do I always forget typehints :D
Thanks for the review @DanielNoord